Common/Brokerages/InteractiveBrokersBrokerageModel.cs

```diff
@@ -62,24 +62,38 @@ public override bool CanSubmitOrder(Security security, Order order, out Brokerag
         {
             message = null;
 
+            // validate security type
+            if (security.Type != SecurityType.Equity &&
+                security.Type != SecurityType.Forex &&
+                security.Type != SecurityType.Option &&
+                security.Type != SecurityType.Future)
```

Comment:
no change required, but typically when there's a set we're comparing against, it's often cleaner to have:
``` csharp
private static readonly HashSet<SecurityType> SupportedSecurityTypes = new HashSet<SecurityType>{ SecurityType.Equity, SecurityType.Forex, SecurityType.Option, SecurityType.Future };

if (!SupportedSecurityTypes.Contains(security.Type)
{
    // error
}
```